components/org.wso2.carbon.deployment.engine/src/main/java/org/wso2/carbon/deployment/engine/internal/DeploymentEngine.java

```diff
@@ -55,9 +55,14 @@
     private RepositoryScanner repositoryScanner;
 
     /**
-     * Repository directory for this deployment engine.
+     * Server repository directory for this deployment engine.
      */
-    private File repositoryDirectory = null;
+    private File serverRepositoryDirectory = null;
```

Comment:
Assigning an object attribute to null is redundant since this is already assigned by default for an object.